### PR TITLE
[MIG] crm_location_nuts: Migration to 12.0

### PR DESCRIPTION
**Pull request by Tardo**
Normal migration

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
It's curious that there's no commit history for 8.0 version

**Comment by Tardo** [comment]
@pedrobaeza ummm i "fixup" some commits but after "Migration to 10.0" commit...

**Comment by pedrobaeza** [comment]
My fault, it was created for that version.

File: crm_location_nuts/models/crm_lead.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def _onchange_country_id(self):
         }
 
     @api.onchange('state_id')
-    def onchange_state_id_base_location_nuts(self):
+    def onchange_state_id_crm_location_nuts(self):
```
Comment (yajo) [inline crm_location_nuts/models/crm_lead.py]: `@onchange()` methods must be private:

```suggestion
    def _onchange_state_id_crm_location_nuts(self):
```

**Review by yajo — CHANGES_REQUESTED** [review]

File: crm_location_nuts/models/crm_lead.py (hunk 1 above)
Comment (Tardo) [reply crm_location_nuts/models/crm_lead.py]: Done, thx!

**Review by yajo — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Code and functional review :+1:

**Pull request merged**